### PR TITLE
Revert "Revert "IE style""

### PR DESCRIPTION
**Pull request by Kumjami**
Reverts numo-labs/isearch-ui#343

### CODE REVIEW

**Comment by Kumjami** [comment]
From:
image
 

To:
image

File: lib/hero-image-header/style.css [hunk 1]
```diff
@@ -71,7 +122,6 @@
 
 .headerContent {
   display: inline-block;
-  margin-top: 30em;
```
Comment (Kumjami) [inline lib/hero-image-header/style.css]: Search bar possition fix.

**Comment by tomgco** [comment]
LGTM!

On a side note @nelsonic it would be awesome to have a source labs bot that did this to our PR's :D

**Comment by nelsonic** [comment]
@tomgco agreed. 👍

**Comment by lennym** [comment]
https://github.com/Huddle/PhantomCSS would probably work as part of the codeship build, but would potentially be completely blown apart by the randomised background images.

**Comment by nelsonic** [comment]
@lennym yeah, that could be a good option, but as you say background images would be an initial obstacle...

**Comment by lennym** [comment]
It looks like it has an option to hide certain selectors before making a comparison, which should help. I'll take a look into it.

**Comment by tomgco** [comment]
> It looks like it has an option to hide certain selectors before making a comparison, which should help. I'll take a look into it.

Cool, that would be sweet.

**Comment by nelsonic** [comment]
@lennym  did a spike to investigate using https://github.com/Huddle/PhantomCSS ...
phantomcss-regression-testing-conclusion

**Pull request merged**